Cargo.toml

```diff
@@ -11,10 +11,10 @@ repository = "https://github.com/maidsafe/bls_dkg"
 version = "0.3.10"
 
 [dependencies]
-aes = "~0.3.2"
+aes = "0.7.4"
```

Comment:
I think we like to use tilde reqs to make updates upstream more flexible.
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements